invisible_cities/reco/xy_algorithms.py

```diff
@@ -150,8 +169,21 @@ def corona(pos, qs,
         # find the SiPMs within new_lm_radius of the new local maximum of charge
         within_new_lm_radius = get_nearby_sipm_inds(new_local_maximum, new_lm_radius, pos, qs)
 
-        # if there are at least msipms within_new_lm_radius, get the barycenter
-        if len(within_new_lm_radius) >= msipm:
+        if masked_sipm is None:
```

Comment:
you can use `()` as the default for `masked_sipm` and avoid these two lines.

Reply:
Probably @jacg can give a more informed opinion here, but googling a bit it seems to me that the `if masked_sipm is None` option is the preferred one for clarity.

Reply:
If the default value of `masked_sipm` is supposed to be an empty *sequence* and you don't rely on any specific list-like behaviour, then using an empty tuple as the default strikes me as the best solution.

The `None`-as-default-arg idiom is standard, but quite ugly. I'm all for avoiding it if it can be avoided elegantly. Empty tuple to represent immutable (constant) empty sequence is perfectly kosher.

Reply:
Ok, then!